edx_proctoring/static/proctoring/js/views/proctored_exam_info.js

```diff
@@ -104,6 +124,13 @@
             }
         },
 
+        isExpiringSoon: function(expirationDate) {
+            var today = new Date();
+            var expirationDateObject = new Date(expirationDate);
+            // Return true if the expiration date is within a month
+            return today > expirationDateObject.getMonth() - 1;
```

Comment:
Thank you for pointing this out - I've changed the comparison to use 28 days in milliseconds, to make it more consistent across the board. This may be changed to a setting in the future.